sample-compose/buildSrc/src/main/java/Configurations.kt

```diff
@@ -1,7 +1,7 @@
 object Flavor {
     const val PRODUCTION = "production"
     const val STAGING = "staging"
-    const val DIMENSIONS = "stage"
+    const val DIMENSION_VERSION = "version"
```

Comment:
@Tuubz Actually, we need to provide a dimension name here instead of a list of dimensions so the config name should be `DIMENSION_[SOMETHING]`. Please check out more detail here https://developer.android.com/reference/tools/gradle-api/7.2/com/android/build/api/dsl/CommonExtension#flavorDimensions().

https://github.com/nimblehq/android-templates/blob/519a71181e06ad9fb5bdcab00c88cc1a1de254fe/sample-compose/app/build.gradle.kts#L60